src/engine/specification.md

```diff
@@ -174,6 +189,38 @@ This structure maps on the [`ExecutionPayload`](https://github.com/ethereum/cons
 - `blockHash`: `DATA`, 32 Bytes
 - `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
 
+### WithdrawalV1
+
+This structure maps onto the validator withdrawal object from the beacon chain spec.
+The fields are encoded as follows:
+
+- `index`: `QUANTITY`, 64 Bits
+- `validatorIndex`: `QUANTITY`, 64 Bits
```

Comment:
```suggestion
- `validator`: `QUANTITY`, 64 Bits
```
Given this is a 64 bit value, "validator" would also clearly refer to a validator index.

Reply:
there are a variety of indices at the CL so generally we are explicit about which ones we are referring to

how important is this change to you?

Reply:
Not super important; wouldn't block over it. I just prefer short, context-descriptive names where possible

Reply:
would prefer validatoIndex (ethereumjs) if its all the same :slightly_smiling_face: